src/lib/extensions/dashboard/DashboardContextObject.ts

```diff
@@ -1,8 +1,10 @@
 import { Params } from '../../models/Params';
 import { ContextObject, isContextObject } from '../Extension';
+import { hubType } from './DashboardExtension';
```

Comment:
Naming convention for types is uppercase for each word so `HubType`

Reply:
Not the biggest fan of adding `Type` to types as seems redundant and if we don't do it everywhere we probably shouldn't do it anywhere. @dev-warner thoughts?

Reply:
yes to upper case, but looks like in the sdk we do use Type as a postfix for Class interfaces